### DISCUSSION TITLE
Load image assets outside of `/public` folder

### DISCUSSION THREAD

**Discussion by softmarshmallow** [vercel/next.js#32592] [Ideas]
### Describe the feature you'd like to request

I'm making a website with landing page & documentation as a monorepo.

I'm centralizing all the docs from several repositories to this main repo so it can be displayed through my website. (The copy & paste process is done by ci/cd; not important)

So the problem is. It seems having all the assets under `/public` seems quite not right to me. The assets are relative to documents, dedicated to specific document.

So I'm having this kind of proj structure.

- docs
  - help
    - image-dedicated-to-help.png
  - tutorials
    - course 1
      - index.mdx
      - image-dedicated-to-course1.png

And I don't want to write scripts to select & copy the *.png files under public, modifying all md/mdx files referencing it AT ALL. (And you'll probably understand why.)

I Hope The Scenario & Problem is Clear.

The repo related to this is - https://github.com/gridaco/grida.co

### Describe the solution you'd like

in `next.config.js`

```
resolve_assets: ["/public", "docs", "other-custom-directory"]
```

### Describe alternatives you've considered

Writing custom scripts, uploading images to s3 and re-writing a mdx - which is totally an overkill.

Splitting to multiple websites & using multizones. (Really no need to go this far)

**Comment by wesbos** [comment]
I'm also hitting this issue - did you ever find a solution?

**Comment by mustaquenadim** [comment]
How people's are handling this issue? I'm not getting any solution. Help me!
